gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.6.2
+version=29.6.3
```

Comment:
Please rebase master, it is 29.6.4 in the master. And update CHANGELOG accordingly.